assets/js/components/surveys/SurveyWizardScheduleStep.jsx

```diff
@@ -99,6 +99,10 @@ class SurveyWizardScheduleStep extends Component {
 
     const { startDate, endDate } = survey.schedule
 
+    const changeTimezone = (timezone) => {
+      dispatch(actions.setTimezone(timezone))
+    }
```

Comment:
This is the first that I've tryed... But, when doing:

```
  changeTimezone(timezone) {
    const { dispatch } = this.props
    dispatch(actions.setTimezone(timezone))
  }
```

I get 

![image](https://user-images.githubusercontent.com/13782680/221582046-1c7b2ffa-761b-4f78-b057-cbe542fdda9b.png)

Because I'm using it as a callback, but not directly calling the function... Any thoughts on this? Is something I'm doing wrong?

Reply:
Yeah, I think this was true in React 0.14 and below but not longer with React 15 or something :shrug: